vlasiator.cpp

```diff
@@ -489,7 +500,9 @@ int main(int argn,char* args[]) {
 		   BgBGrid,
 		   volGrid,
 		   technicalGrid,
-		   sysBoundaries, 0.0, 1.0
```

Comment:
Greetings from the random indentation service. ;)

Reply:
VScode does't show this indentation. But it is there I can see it in Vim.